eng/pipelines/templates/jobs/vmr-build.yml

```diff
@@ -184,6 +184,9 @@ jobs:
   - name: artifactsStagingDir
     value: $(Build.ArtifactStagingDirectory)/artifacts
 
+  - name: artifactsPackagesDir
+    value: $(Build.ArtifactStagingDirectory)/packages
```

Comment:
Suggest for future changes that we put this under $(Build.ArtifactStagingDirectory)/artifacts/packages.

Reply:
This folder is the "pre-published" assets. These aren't split by blob vs package, they're the raw layout we have before we actually run publish.

I'll change the folder name to better describe that.

Reply:
Sounds good. My intention here is that publish.proj gets run, sets the various non-shipping, shipping, etc. dirs under a common parent (e.g. ArtifactsStagingDir/publish or /artifacts), then we use the pipeline output: to publish this entire thing as one artifact.

Reply:
Yeah this folder is for assets to be published (before publish.proj runs). The idea is that if we fail, we publish these assets as we almost assuredly didn't run Publish.proj before we failed.